k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/fluent-bit/fluent-bit.yaml

```diff
@@ -206,7 +206,7 @@ data:
     [FILTER]
         Name                aws
         Match               host.*
-        imds_version        v1
+        imds_version        v2
```

Comment:
To access IMDSv2 with the default hop limit of 1, does the daemonset of FLB need to run in the host network?

Reply:
If the hop limit is set to 1, then, yes, the pod would have to use host networking. The default on modern EKS AMIs is set to 2, though.

Reply:
Yeah from what I've seen, managed EKS nodes are configured with hop limit of 2 by default. This doesn't account for self-managed nodes unfortunately.